pom.xml

```diff
@@ -153,6 +153,13 @@
 
   <dependencyManagement>
     <dependencies>
+      <dependency>
+        <groupId>org.kie</groupId>
+        <artifactId>kie-platform-bom</artifactId>
+        <version>${version.org.kie}</version>
+        <type>pom</type>
+        <scope>import</scope>
+      </dependency>
```

Comment:
Since this imports all our BOMs, you can delete then `jbpm-bom, drools-bom, kie-soup-bom, kie-dmn-bom` from here. I think the same can be done for all other (drools, droolsjbpm-integration...) repositories.

Reply:
OK! Remobed these duplicated boms.